frame/assets/src/lib.rs

```diff
@@ -153,6 +154,9 @@ pub trait Config: frame_system::Config {
 
 	/// Weight information for extrinsics in this pallet.
 	type WeightInfo: WeightInfo;
+
+	/// Type parameter for ERC20 compatible
+	type IsERC20Compatible: Get<bool>;
```

Comment:
```suggestion
	type AllowApprovalSpending: Get<bool>;
```

This is my suggestion on naming... not a big deal i guess, but this is more clear to me.

Reply:
To be honest it might be nice to actually have configuration flags on each of these kind of unique features, like Freezing assets or whatever.

I know in the ethereum space, they make it really flexible how you can customize the use of your token.